Editor/Window/ManagedEC2Deployment.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Threading.Tasks;
+using AmazonGameLift.Editor;
+using UnityEngine;
+
+namespace AmazonGameLift.Editor
+{
+    public class ManagedEC2Deployment
+    {
+        private readonly DeploymentSettings _model;
+        private readonly FleetParameters _parameters;
+        
+        internal ManagedEC2Deployment(DeploymentSettings model, FleetParameters parameters)
+        {
+            _model = model;
+            _parameters = parameters;
+        }
+        
+        public void UpdateModelFromParameters()
+        {
+            _model.FleetName = _parameters.FleetName;
+            _model.BuildName = _parameters.BuildName;
+            _model.LaunchParameters = _parameters.LaunchParameters;
+            _model.BuildFolderPath = _parameters.GameServerFolder;
+            _model.BuildFilePath = _parameters.GameServerFile;
+            _model.BuildOperatingSystem = _parameters.OperatingSystem;
+        }
+        
+        public void StartDeployment()
+        {
+            UpdateModelFromParameters();
```

Comment:
_parameters is passed in through the constructor, which means the `_parameters` never change after initialization? Should you be passing in `parameters` to the `UpdateModelFromParameters` instead?

Reply:
It would work as its a reference type, but I have changed it to make more sense